src/components/Radiobutton.js

```diff
@@ -13,22 +13,18 @@ export const Radiobutton = ({ title, disabled, selected, onPress, containerStyle
         ? texts.accessibilityLabels.checkbox.active
         : texts.accessibilityLabels.checkbox.inactive
     } (${title})`}
-    title={title}
     checked={selected}
+    checkedIcon={<Icon.RadioButtonFilled size={22} style={styles.rightContentContainer} />}
+    containerStyle={[styles.containerStyle, containerStyle]}
+    disabled={disabled}
```

Comment:
past surveys are not answered via `Touchable` because `Tocuhable` has a `disabled` prop. But with `RadioButton` you can click on the options in the past surveys, but the reply button does not appear. so I added `disabled` prop to `RadioButton`. `disabled` prop was previously only used for styling.